alerting/claranet-ingest/variables.tf

```
@@ -0,0 +1,32 @@
+# Global
+
+variable "suffix" {
+  description = "Optional suffix for the integration name"
+  type        = string
+  default     = ""
+}
+
+# Ingest-Alerting Integration specific
+
+variable "enabled" {
+  description = "Whether the Webhook integration is enabled"
+  type        = bool
+  default     = true
+}
+
+variable "url" {
+  description = "The ingest-alerting URL to use"
+  type        = string
+  default     = "https://ingest-alerting.fr.clara.net/splunk"
```

```suggestion
  default     = "https://ingest-alerting.fr.clara.net/datadog"
```
il faut pas mettre ça ?

---

Fixed!